server/src/cli.rs

```diff
@@ -10,25 +10,30 @@ pub enum EdgeMode {
     Offline(OfflineArgs),
 }
 
-pub enum EdgeArg {
+pub enum Persister {
     Redis(String),
-    InMemory,
+    File(PathBuf),
+    None,
 }
 
-impl From<EdgeArgs> for EdgeArg {
+impl From<EdgeArgs> for Persister {
     fn from(value: EdgeArgs) -> Self {
         if let Some(redis_url) = value.redis_url {
-            return EdgeArg::Redis(redis_url);
+            return Persister::Redis(redis_url);
         };
 
-        EdgeArg::InMemory
+        if let Some(backup_folder) = value.backup_folder {
+            return Persister::File(backup_folder);
+        };
+
+        Persister::None
     }
```

Comment:
This can be yeeted instead (done). The work in the builder is correct, this is here because this was the start of a thought and it's a public trait so we never got a warning it was unused. Good spot!